packs/spell-effects/spell-effect-vital-beacon.json

```diff
@@ -15,7 +15,7 @@
             "value": 4
         },
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Vital Beacon]<br />Vitality radiates outward from you, allowing others to supplicate and receive healing. Once per round, either you or an ally can use an Interact action to supplicate and lay hands upon you to regain Hit Points. Each time the beacon heals someone, it decreases in strength. It restores 4d10 Hit Points to the first creature, 4d8 Hit Points to the second, 4d6 Hit Points to the third, and 4d4 Hit Points to the fourth, after which the spell ends. You can have only one vital beacon active at a time.<br />@Damage[(@item.level)d10[healing]]{First Activation (d10)}<br />@Damage[(@item.level)d8[healing]]{Second Activation (d8)}<br />@Damage[(@item.level)d6[healing]]{Third Activation (d6)}<br />@Damage[(@item.level)d4[healing]]{Fourth Activation (d4)}</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Vital Beacon]</p>\n<p>Once per round, either you or an ally can use an Interact action to supplicate and lay hands upon you to regain Hit Points. It restores @Damage[(@item.level)d10[healing,vitality]|shortLabel] Hit Points to the first creature, @Damage[(@item.level)d8[healing,vitality]|shortLabel] Hit Points to the second, @Damage[(@item.level)d6[healing,vitality]|shortLabel] Hit Points to the third, and @Damage[(@item.level)d4[healing,vitality]|shortLabel] Hit Points to the fourth, after which the spell ends.</p>"
```

Comment:
We can maybe justify the effect by using a counter? 2 + 2*badge value gives the die size, can we use that in the formula to roll so there's just one formula and people can track it counting down from 4 to 1?

Reply:
Wait, die size is resolvable?

Reply:
`@Damage` does a better job of resolving than other inlines. Even the damage type is resolvable.

Reply:
Though, it has to use `@` notation, not `{}` which is awkward but 🤷

Reply:
I knew about the damage type but it never even occurred to me to try with die sizes. That's neat, I'll add that later then!